src/views/controllers/Staff.vue

```diff
@@ -55,20 +66,27 @@ f<template>
 				</div>
 			</div>
 		</div>
+		<!-- END STAFF ROW -->
```

Comment:
Can this comment be moved to before line 66? This comment is several div's outside of the first comment.